roles/firewall/tasks/main.yml

```diff
@@ -1,31 +1,34 @@
 ---
 - name: Install packages
-  apt: name=iptables-persistent
+  apt: name=iptables
```

Comment:
Der hat IMO nichts in den Roles zu suchen, das sollte ein einzelner ansible-Aufruf sein.